li-utils/src/main/javaPegasus/com/linkedin/common/urn/RoleUrn.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.common.urn;
+
+import com.linkedin.data.template.Custom;
+import com.linkedin.data.template.DirectCoercer;
+import com.linkedin.data.template.TemplateOutputCastException;
+
+import java.net.URISyntaxException;
+
+
+public final class RoleUrn extends Urn {
```

Comment:
This class is not necessary, feel free to remove!

Reply:
Done